scripts/ci/check-npm-dependencies.ts

```diff
@@ -16,15 +16,17 @@ const packagesPath = path.join(rootPath, 'packages');
 const args = process.argv.slice(2);
 
 if (args.length < 2)
-    throw new Error('Usage: yarn tsx check-npm-dependencies.ts <package_name> <semver>');
+    throw new Error('Usage: yarn tsx check-npm-dependencies.ts <packageName> <semanticVersion>');
 
-const [packageName, semver] = args;
+const [packageName, semanticVersion] = args;
```

Comment:
Changing names in this files since the ts rules applied in this path are complaining about it having same name as the package `semver` that is used in imported helper.